content/source/docs/cloud/vcs/github-app.html.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: "cloud"
+page_title: "GitHub.com (GitHub App) - VCS Providers - Terraform Cloud"
+---
+
+[private module registry]: ../registry/index.html
+[policy sets]: ../sentinel/manage-policies.html
+[vcs settings]: ../workspaces/vcs.html
+[create]: ../workspaces/creating.html
+[owners]: ../users-teams-organizations/teams.html#the-owners-team
+
+# Configuration-Free GitHub Usage
+
+These instructions are for using repositories from GitHub.com with Terraform Cloud workspaces, without configuring an OAuth connection.
+
+Using GitHub this way does not require any action from an [organization owner][owners]; anyone who can create workspaces or administer existing workspaces can connect GitHub repositories.
+
+This method uses a preconfigured GitHub App, and only works with GitHub.com. There are separate instructions for connecting to [GitHub.com via OAuth](./github.html), connecting to [GitHub Enterprise](./github-enterprise.html), and connecting to [other supported VCS providers.](./index.html)
+
+## Using GitHub Repositories
+
+You can connect GitHub repositories to Terraform Cloud workspaces without changing any organization settings.
+
+To do so, choose "GitHub.com" on the "Connect to a version control provider" screen, which is shown when [creating a new workspace][create] or [changing a workspace's VCS connection][vcs settings]. Authorize access to GitHub if necessary. On the next screen, select a GitHub account or organization from the drop-down menu (or add a new organization) and choose a repository from the list.
+
+The controls on the "Connect to a version control provider" screen can vary, depending on your permissions and your organization's settings:
+
+- In organizations with no VCS connections configured:
+    - [Owners][] will see several drop-down menus, sorted by product family. Choose "GitHub.com" (_not_ "GitHub.com (Custom)") from the GitHub menu.
+    - Non-owners will see a "GitHub" button.
+- In organizations with an existing VCS connection, only the connected providers are shown. Click the "Connect to a different VCS" link to reveal the provider menus (owners) or the GitHub button (non-owners).
+
+![Screenshot: the "Connect to a version control provider" screen of the "Create a new workspace" page.](./images/gh-app-choose-provider.png)
+
+![Screenshot: Terraform Cloud's repository list, with the organization switcher menu open and the "Add another organization" option visible.](./images/gh-app-install.png)
+
+## GitHub Permissions
+
+Unlike OAuth connections, the Terraform Cloud GitHub App does not configure a Terraform Cloud organization to always act as one particular GitHub user. Instead, each Terraform Cloud user authenticates individually, and can use GitHub resources within Terraform Cloud according to their own GitHub organization memberships and access permissions.
```

Comment:
There are a ton of negatives in this first sentence and I found it a bit hard to follow. Can we phrase it positively instead, or put the second sentence before the first?

Reply:
Hmm, you're right. I'll do both, probably.

Reply:
Shrinking that first sentence and moving it down into a "-> note".